src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import Button from 'react-bootstrap/Button';
```

Comment:
We need to maintain ladder structure for imports.

Let's separate installed package 1st, custom component 2nd and css and images 3rd by newlines. 

Let's make those changes from the next PR's

Reply:
sure, forgot about that.